.github/workflows/node.js.yml

```diff
@@ -68,6 +69,12 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y gettext
 
+    - name: Clone the detailed impacts
+      run: |
+        eval `ssh-agent -s`
+        ssh-add - <<< '${{ secrets.SCALINGO_SSH_KEY }}'
```

Comment:
I might be confused but where does this `secrets.SCALINGO_SSH_KEY` value come from, especially as we're in a github action and see it defined nowhere? In any case it might be worth adding a comment here and/or an entry in the README about it.

Reply:
It's defined in the Github repo secrets but sure, I'll add a comment :+1:

Reply:
Bat as it's not a private key used only for Scalingo, I should rename it to clarify things